packages/docs/content/getting-started/for-developers.mdx

```diff
@@ -0,0 +1,384 @@
+
+
+## Including the CSS
+
+No matter what framework you use to build your application, you will need to include the design system styles in order for your application to look the way it should.
+
+### With HTML
+
+The simplest way to do that is to include a resource link element in your HTML document, like this:
+
+```html
+<head>
+  <link rel="stylesheet" href="https://design.cms.gov/cdn/design-system/5.0.0/css/index.css" />
+</head>
+```
+
+If you are getting the files from npm, you will want to change the `href` to reference the files from your local `node_modules` folder. Likewise if you've downloaded the files directly, the `href` will point to wherever you've put your files.
+
+### With an asset bundler
+For projects that use an asset bundler, include the assets in whatever way your bundler recommends. For instance, in a WebPack or create-react-app project, you might include the styles in your CSS like this:
+
+```css
+@import '@cmsgov/design-system/dist/css/index';
+```
+
+Or you might import it from your JavaScript like this:
+
+```css
+import '@cmsgov/design-system/dist/css/index.css';
+```
+
+### Fonts and images
+
+Note that our stylesheets also reference fonts and images, so you will need to make sure these assets (`dist/fonts` and `dist/images`) are available to your site/application at the correct relative paths. You don't need to think about this if you're loading the stylesheet from the CDN, and most asset bundlers will be able to locate these additional assets without additional configuration.
+
+## Using the CSS
+
+The design system styles are broken down into a few categories:
+
+1. Base and typography styles
+2. Layout styles
+3. Component styles
+4. Utility classes
+
+### Base and typography styles
+
+For all the other styles to inherit the correct typography properties, the root of your page content should have a `ds-base` class applied to it. Please see [our base styles documentation](/foundation/base) for more details. Additional [heading](foundation/typography/headings) and [body](foundation/typography/body) typography classes can be found on their respective documentation pages.
+
+### Layout styles
+
+The design system provides a number of layout classes to implement visual structure in your application. Please see [the layout grid documentation](/foundation/layout-grid/layout-grid) to get started. Further resources can be found in the [Layouts](/layouts) section of the doc site.
+
+### Component styles
+
+See the [Using the components]() section below.
+
+### Utility classes
+
+Utility classes help you apply single CSS properties to elements that follow design-system styles. See the [Utilities](/utilities) section of the doc site for more details.
```

Comment:
Should design-system have a hyphen?

Reply:
I believe so technically because "design system" is being used as an adjective to describe "styles", but we're inconsistent on this, so I can leave the hyphen out.